requirements_test_unit.txt

```diff
@@ -1,3 +1,4 @@
 pytest
 responses
 scikit-learn==1.3.2
+openai
```

Comment:
Consider pinning to specific version of `openai`

Reply:
Will do